src/customJs/subscribeTool.js

```diff
@@ -50,6 +54,7 @@ unlayer.registerTool({
       },
     }),
     exporters: {
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
       email: function () {},
```

Comment:
Wue opinás de poner un `//TODO` y un `console.log` aquí?

```suggestion
      // eslint-disable-next-line @typescript-eslint/no-empty-function
      email: function () {
        // TODO: implement this function
        console.log(['Email exporter', ...arguments]);
      },
```

Reply:
🚀